install/check-minimum-requirements.sh

```diff
@@ -9,6 +9,12 @@ if docker compose version &>/dev/null; then
 else
   # Do NOT use $dc instead of `docker-compose` below as older versions don't support certain options and fail
   COMPOSE_VERSION=$(docker-compose --version | sed 's/docker-compose version \(.\{1,\}\),.*/\1/')
+
+  # Another check if is the newer version with additional prefix
+  if [[ $COMPOSE_VERSION =~ v ]]; then
+    # Extract only the version number
+    COMPOSE_VERSION=$(echo ${COMPOSE_VERSION} | rev | cut -d 'v' -f 1 | rev)
```

Comment:
What do you think about this?
```suggestion
    COMPOSE_VERSION=$(echo ${COMPOSE_VERSION} | sed 's/^v//')
```

Reply:
@aminvakil doesn't quite work. The content is `version v2.0.1`. That `v` is the last one.

The full output is:
```bash
[luiz@fedora self-hosted]$ docker-compose -v
Docker Compose version v2.0.1
```

So my strategy was ignoring 100% of what's before `v`, because doesn't matter if they change the prefix again in the future.

What you can do if you want a cleaner approach is to ignore the entire `Docker Compose version v` prefix.

Reply:
Per my comment on the issue about #1116 , the scripts checks for `docker-compose` version using `docker compose version --short` which returns:
```
$ docker compose version --short
2.2.0
```
They changed lots of things even after `2.0.0`, I cannot test using versions > `2` < `2.2.0` right now, but it drops the `v` prefix already here:
`COMPOSE_VERSION=$(docker compose version --short | sed 's/v\{0,1\}\(.\{1,\}\)/\1/')`

Reply:
@aminvakil Well, this is not the current code. The `docker compose version [--short]` command repeats 3 times:

https://github.com/getsentry/self-hosted/pull/1177#diff-1b2cb955f165ce00dbc0caa8eb2f9f54032b41f6f52d144f9a1313f53a8b5e1fR10-R11

I'll try to change all of them.

Reply:
Please read https://github.com/getsentry/self-hosted/pull/1116#issuecomment-944900571 completely. As once mentioned to me :) setting `docker-compose` is not mandatory anymore.
And we don't want to change the behaviour which was working for `docker-compose` < 2.
Also I'm not sure if `--short` existed for `docker-compose` < 2.

Reply:
@aminvakil sorry. I just read it in the entire issue and I don't know what to do.
The first solution solved the problem and my second attempt broke some behavior.
I also don't know the `docker-compose` internals.

I'll wait for some recommendations. 

My recommendation still is the first one. Fix the problem without compromising the initial asserts.

Reply:
Reverted again to the initial PR state.

Reply:
The challenge is that, while `docker compose` is guaranteed(?) to be v2, `docker-compose` can be either v1 or v2. I have some ideas ... going to post back on #1132.

Reply:
Hello @chadwhitacre. Now I understood the problem but I can't understand why this current PR state isn't enough.
It solves all scenarios that I can think about.

Reply:
@thenets I feel bad for turning down your contribution ... how about we merge this one _and_ #1179 (which refactors `check-minimum-requirements.sh` for legibility, and also addresses additional instances of `docker-compose` throughout the codebase)? 😁

Reply:
@chadwhitacre no problem at all. As I said in the beginning, probably exists some better way to solve the problem. You can discard my PR, no problem.
But I still think that until now, my PR is the only solution for my specific case without compromising existent scenarios.